hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -2271,6 +2278,16 @@ public void testListBucket()
       volABucketCount++;
     }
     Assert.assertEquals(20, volABucketCount);
+
+    Iterator<? extends OzoneBucket> volABucketHasSnapshotIter =
+        volA.listBuckets("bucket-", null, true);
+    int volABucketHasSnapshotCount = 0;
+    while (volABucketHasSnapshotIter.hasNext()) {
+      volABucketHasSnapshotIter.next();
+      volABucketHasSnapshotCount++;
+    }
+    Assert.assertEquals(20, volABucketHasSnapshotCount);
+
```

Comment:
This is repetitive code. You can create helper function to fetch/count buckets in the volume and assert that.

```
  Assert.assertEquals(20, countBuckets(volA, "bucket-", null, true));

  // Helper function
  private int countBuckets(OzoneVolume volume,
                           String bucketPrefix,
                           String continuationToken,
                           boolean hasSnapshot) {
    Iterator<? extends OzoneBucket> bucketIterator =
        volume.listBuckets(bucketPrefix, continuationToken, hasSnapshot);
    int bucketCount = 0;
    while (bucketIterator.hasNext()) {
      bucketIterator.next();
      bucketCount++;
    }
   return bucketCount;
  }
```